image_view/CMakeLists.txt

```
@@ -39,10 +39,16 @@ if(ANDROID)
 endif()
 
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(GTK3 REQUIRED gtk+-3.0)
+pkg_check_modules(GTK3 gtk+-3.0)
+if(NOT GTK3_FOUND)
+  message(STATUS "GTK3 is not found. Some features are disabled.")
+endif()
 include_directories(${GTK3_INCLUDE_DIRS})
```

I recommend the following:

```suggestion
pkg_check_modules(GTK3 gtk+-3.0)
if(GTK3_FOUND)
  include_directories(${GTK3_INCLUDE_DIRS})
else()
  message(STATUS "GTK3 is not found. Some features are disabled.")
endif()
```